README.md

```diff
@@ -52,6 +52,12 @@ When `true`, the plugin will attempt to categorize renditions by lines of horizo
 
 When `true`, the plugin will attach bitrate information to the resolution labels (e.g. `720p @ 13806 kbps`). This option has no effect when `useResolutionLabels` is `false` or when resolution information is unavailable.
 
+### `measureShortEdge`
+
+> Type: `boolean` Default: `false`
+
+When `true`, the plugin will use a short edge to display the resolution for vertical video.
```

Comment:
```suggestion
When `true`, the plugin will display the length of the shorter edge as the resolution, e.g. the width of a vertical video. Otherwise the height is used.
```